packaging/installer/kickstart.sh

```diff
@@ -1217,14 +1227,14 @@ set_auto_updates() {
     # This first case is for catching using a new kickstart script with an old build. It can be safely removed after v1.34.0 is released.
     elif ! grep -q '\-\-enable-auto-updates' ${updater}; then
       echo
-    elif ! ${ROOTCMD} ${updater} --enable-auto-updates "${NETDATA_AUTO_UPDATE_TYPE}"; then
+    elif ! run_as_root ${updater} --enable-auto-updates "${NETDATA_AUTO_UPDATE_TYPE}"; then
       warning "Failed to enable auto updates. Netdata will still work, but you will need to update manually."
```

Comment:
Hello @Ferroin ,

`shellcheck` is reporting issues in the modified lines, are you going to address these issues?

best regards!